arviz/stats/density_utils.py

```diff
@@ -436,6 +440,7 @@ def kde(x, circular=False, **kwargs):
         >>> plt.show()
 
     Default density estimation for linear data with custom limits
+
     .. plot::
         :context: close-figs
     # It accepts tuples and lists of length 2.
```

Comment:
This isn't rendering correctly. All directives need a blank line before and after, and the comment should be indented as it should be treated as code.